### DISCUSSION TITLE
now/

### DISCUSSION THREAD

**Discussion by giscus** [residualsun1/Residualsun#44] [General]
# now/

My Blog

https://guozheng.rbind.io/now/

**Comment by earfanfan** [comment]
哇嗷，这些书的名字我竟然从未听闻，都是老师们给你推荐的吗？

**Reply by residualsun1** [reply]
这些基本是民族志，《努尔人》、《缅甸高地诸政治体系》和《摩洛哥田野作业反思》是老师推荐的，其他的就是自己去找的啦。
